src/utils/utils.js

```diff
@@ -226,8 +227,8 @@ module.exports = {
   * Returns the path where the blockchain log directory is, and also creates the directory if it doesn't exist.
   */
   getLogDir: () => {
-    const versionDir = getVersionDir();
-    const logDir = `${versionDir}/logs`;
+    const osDataDir = app.getPath('userData');
+    const logDir = `${osDataDir}/logs/${version}`;
```

Comment:
This is the old path. Livia made a change to put it in the same dir as the version dir. The fix should be for making it work with `getVersionDir()`.